src/Symfony/Bundle/WebProfilerBundle/Controller/ProfilerController.php

```diff
@@ -162,6 +162,27 @@ public function toolbarAction(Request $request, ?string $token = null): Response
         ]);
     }
 
+    /**
+     * Renders the Web Debug Toolbar stylesheet.
+     *
+     * @throws NotFoundHttpException
+     */
+    public function toolbarStylesheetAction(): Response
+    {
+        $this->denyAccessIfProfilerDisabled();
+
+        $this->cspHandler?->disableCsp();
+
+        return new Response(
+            $this->twig->render('@WebProfiler/Profiler/toolbar.css.twig'),
+            200,
+            [
+                'Content-Type' => 'text/css',
+                'Cache-Control' => 'max-age=3600, private',
```

Comment:
nit picking here but caching could cause small issues during upgrades
if we think this is important (maybe it's not), a solution could be doing some cache validation?
is so the kernel version (`Symfony\Component\HttpKernel\Kernel::VERSION`) could be a nice candidate